### PR TITLE
fix: apply checkstyle fixes to aqa tests

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

**Review by palakpatel311 — APPROVED** [review]

File: src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package gov.healthit.chpl.aqa.api.payLoad;
+
+import gov.healthit.chpl.aqa.api.asserts.ComplaintsAsserts;
+
+public final class ComplaintsPayload {
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java]: A class with the name "ComplaintsPayload" should be in a file named "ComplaintsPayload.java". This class was originally named "complaintsPayload", which doesn't meet our checkstyle rules. This file should be removed

File: src/test/java/gov/healthit/chpl/aqa/asserts/ListingHistoryEyeAsserts.java [hunk 2]
```diff
@@ -54,7 +54,8 @@ public void iSeeListingHistory(final String expectedEditDetails) throws Exceptio
         }
         Date actualDate = getFormatedDateTimeZone(latestDate);
         Date currentDate = new Date();
-        assertTrue(((currentDate.getTime() - actualDate.getTime()) < TIME_DURATION), "Listing history is missing for today's date");
+        assertTrue(((currentDate.getTime() - actualDate.getTime()) < TIME_DURATION),
+                "Listing history is missing for today's date");
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/asserts/ListingHistoryEyeAsserts.java:58]: I know you didn't change the contents, but the text here doesn't match the assertion

**Review by andlar — CHANGES_REQUESTED** [review]

File: src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java (hunk 1 above)
Comment (ashwinimore) [reply src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java]: I don't see that file on my local anymore, how to fix this?

Comment (ashwinimore) [reply src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java]: removed it

**Review by andlar — APPROVED** [review]

**Pull request merged**